buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/PublishingConfiguration.kt

```diff
@@ -134,6 +136,13 @@ private fun Project.configureSigning() {
         logger.lifecycle("The following publications are getting signed: ${extensions.getByType<PublishingExtension>().publications.map { it.name }}")
         sign(*extensions.getByType<PublishingExtension>().publications.toTypedArray())
     }
+
+    tasks.withType<PublishToMavenRepository>().configureEach {
```

Comment:
publishToMavenLocal also have type `PublishToMavenRepository`?

Reply:
Yes, I believe all `publish*To*Repository` are registered in a similar way.